### PR TITLE
Added additional documentation on APIs

### PR DESCRIPTION
**Pull request by arueth**
Can be viewed here: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/rueth-saxml-httpserver-docs/saxml-on-gke/httpserver/README.md

### CODE REVIEW

**Comment by arueth** [comment]
@vivianrwu  could you take a look at this?

File: saxml-on-gke/httpserver/README.md [hunk 1]
```diff
@@ -34,139 +34,342 @@ docker run -e SAX_ROOT=gs://${GSBUCKET}/sax-root -p 8888:8888 -it sax-http
...
+
+```
+{
+    "model": <String>,
+    "query": <String>,
+    "extra_inputs": {
+        "temperature": <Number>,
+        "per_example_max_decode_steps": <Number>,
+        "per_example_top_k": <Number>,
+        "per_example_top_p": <Number>
```
Comment (vivianrwu) [inline saxml-on-gke/httpserver/README.md:177]: temperature: float,
per_example_max_decode_steps: int,
per_example_top_k: int,
per_example_top_p: float

File: saxml-on-gke/httpserver/README.md [hunk 2]
```diff
@@ -34,139 +34,342 @@ docker run -e SAX_ROOT=gs://${GSBUCKET}/sax-root -p 8888:8888 -it sax-http
...
+    ],
+    ...
 ]
 ```
 
-### List a Sax Cell
+`[[<String>, <Number>]]` is an array of arrays
+
+- `<String>` is the response from the model.
+- `<Number>` is the score of the response.
```
Comment (vivianrwu) [inline saxml-on-gke/httpserver/README.md:207]: float

File: saxml-on-gke/httpserver/README.md [hunk 3]
```diff
@@ -34,139 +34,342 @@ docker run -e SAX_ROOT=gs://${GSBUCKET}/sax-root -p 8888:8888 -it sax-http
...
-  "extra_inputs": {
-    "temperature": 0.9,
-    "per_example_max_decode_steps": 128,
-    "per_example_top_k": 1,
-    "per_example_top_p": 1.0
-  }
+    "model": <String>,
+    "model_path": <String>,
+    "checkpoint": <String>,
+    "replicas": <Number>
```
Comment (vivianrwu) [inline saxml-on-gke/httpserver/README.md:288]: ditto

File: saxml-on-gke/httpserver/README.md [hunk 4]
```diff
@@ -34,139 +34,342 @@ docker run -e SAX_ROOT=gs://${GSBUCKET}/sax-root -p 8888:8888 -it sax-http
...
+}
+```
+
+- `model` is the name of the model unpublished.
+
+### update
+
+`/update` is used to updated an existing model.
+
+#### update Request
```
Comment (vivianrwu) [inline saxml-on-gke/httpserver/README.md:341]: same in this section, can change the numbers to int so user is clear

**Comment by arueth** [comment]
I can add the types to the descriptions, but in JSON there is just Number and not "sub types" of int or float.

**Review by vivianrwu — APPROVED** [review]

**Pull request merged**